httpdocs/js/webots-cloud.js

```diff
@@ -164,29 +178,25 @@ document.addEventListener('DOMContentLoaded', function() {
       const words = data.url.substring(19).split('/');
       const repository = `https://github.com/${words[0]}/${words[1]}`;
       const animation = `https://${words[0]}.github.io/${words[1]}/${words[3]}`;
+      const title = data.title === '' ? '<i>anonymous</i>' : data.title;
       const updated = data.updated.replace(' ',
-        `<br><i class="is-clickable fas fa-sync" id="sync-${data.id}" data-url="${data.url}" title="Re-synchronize now"></i> `
+        `<br><i style="color:grey" class="is-clickable far fa-trash-alt" id="delete-${data.id}" title="Delete '${title}' simulation"></i> `
       );
-      const title = data.title === '' ? '<i>anonymous</i>' : data.title;
       let type;
-      let competitors;
       if (data.type == 'demo') {
-        type = `<i class="fas fa-grin-stars fa-lg" title="${data.type}"></i>`;
-        competitors = '';
+        type = `<i class="fas fa-chalkboard-teacher fa-lg" title="${data.type}"></i>`;
       } else {
-        const icon = (data.type == 'benchmark') ? 'award' : 'trophy';
+        const icon = (data.type == 'benchmark') ? 'chart-line' : 'question';
```

Comment:
I like the char-line icon for benchmarks, but I would like to keep the trophy for competitions, as if it is not a benchmark, it is a competition.
```suggestion
        const icon = (data.type == 'benchmark') ? 'chart-line' : 'trophy';  // if not a benchmark, it is a competition
```